model/lifecycle.go

```diff
@@ -248,14 +224,10 @@ func SetBuildPriority(buildId string, priority int64, caller string) error {
 		if err != nil {
 			return errors.Wrapf(err, "can't get tasks for build '%s'", buildId)
 		}
-		var deactivatedTasks []task.Task
-		deactivatedTasks, err = task.DeactivateTasks(tasks, caller)
+		err = task.DeactivateTasks(tasks, caller)
```

Comment:
nit: now that this has only an error to return can we combine with with the if statement